### PR TITLE
feat: submit bet on press enter key

### PR DESCRIPTION
**Pull request by creisle**

### CODE REVIEW

File: src/components/BettingDialog/index.tsx [hunk 1]
```diff
@@ -28,11 +28,18 @@ function BettingDialog(props: BettingDialogProps) {
...
         setBet(value ? Number(value) : null);
-    }
+    };
 
     const handleBetPlaced = () => {
         onBetPlaced(bet as number);
-    }
+    };
+
+    const handleOnKeyPress = (event: any) => {
```
Comment (areisle) [inline src/components/BettingDialog/index.tsx]: event should have a type

File: src/components/BettingDialog/index.tsx [hunk 2]
```diff
@@ -28,11 +28,18 @@ function BettingDialog(props: BettingDialogProps) {
...
+        if (event.charCode === 13) {
+            event.preventDefault();
```
Comment (areisle) [inline src/components/BettingDialog/index.tsx]: I don't think this is necessary?

Comment (creisle) [reply src/components/BettingDialog/index.tsx]: removed

File: src/components/BettingDialog/index.tsx (hunk 1 above)
Comment (creisle) [reply src/components/BettingDialog/index.tsx]: fixed

File: src/components/BettingDialog/index.tsx [hunk 3]
```diff
@@ -54,6 +61,7 @@ function BettingDialog(props: BettingDialogProps) {
                         max,
                         min: 0,
                     }}
+                    onKeyPress={handleOnKeyPress}
```
Comment (areisle) [inline src/components/BettingDialog/index.tsx:63]: this should be on the input not the dialog

**Review by areisle — APPROVED** [review]

**Pull request merged**